cognite/client/_api_client.py

```diff
@@ -587,7 +587,7 @@ def _aggregate(
     @overload
     def _create_multiple(
         self,
-        items: Union[Sequence[T_CogniteResource], Sequence[Dict[str, Any]]],
+        items: Union[Sequence[CogniteResource], Sequence[Dict[str, Any]]],
```

Comment:
Why these changes away from `T_*`?

Reply:
There was an issue, which I cannot remember any more, and I cannot recreate it. So reverted this one.